codegen/project.go

```diff
@@ -336,10 +337,12 @@ func (p *Project) GetRustOutputProtobufPath() string {
 }
 
 func (p *Project) ChainEndpoint() (string, error) {
-	if ChainConfigByID[p.Network] == nil {
+	networks := networks.GetRegistryNetworksWithSubstreams()
+	net := networks[p.Network]
```

Comment:
Do we want to use `Find` here or do we assume id is correct?

Reply:
Yeah use Find